python/ray/util/sgd/torch/training_operator.py

```diff
@@ -777,7 +777,15 @@ def from_ptl(cls,
                  lightning_module_cls,
                  train_dataloader=None,
                  val_dataloader=None):
-        """Creates a TrainingOperator from a Pytorch Lightning Module.
+        """A utility method to create a custom TrainingOperator class from a
+        Pytorch Lightning Module.
```

Comment:
pydoc 1 line summaries should only be 1 line

Reply:
done